pkgs/test/mono_pkg.yaml

```diff
@@ -1,11 +1,15 @@
 dart:
- - dev
+  - dev
```

Comment:
Why was this section removed in `test_api` but not here?

Reply:
Weirdness. `mono_repo` doesn't have support for specifying the SDK in a stage w/out a group – that's the short version.